### PR TITLE
update persistence

### PR DESCRIPTION
**Pull request by CarloLucibello**
- add to convenience function,  `readgraph` and `writegraph` for reading/writing a single graph from a file (the first) without going through a dict. Possible alternatives `loadgraph` and `savegraph`. I choose the former because `read` and `write` came first to my mind when I was looking for i/o functions, then I had to recur to the docs to find `load` and `save`, but that's just me maybe

- AbstractString -> String

### CODE REVIEW

**Comment by sbromberger** [comment]
We should be consistent: either `load / loadgraph / save / savegraph` or `read / readgraph / write / writegraph`. Other than that, I guess this is ok, though see #294 and #258.

**Comment by CarloLucibello** [comment]
pfff, I have the memory of a red fish. 

Still I'm not satisfied with any option at all, mainly I don't like such generic names  as `load("file"::String)` or `read("file"::String)`  for signatures not  specializez by the Graph type

What about `loadgraph` and `loadgraphs`?  or `readgraph` and `readgraphs`? Just brainstorming, I don't know if I like this either :)

In any case if none of us comes with a strong opinion I'll go with the conservative/nonbreaking option `load` and `loadgraph` for the time being

**Comment by sbromberger** [comment]
HAH :)

OK, I'm good with whatever for the time being. I think persistence needs another look at some point. I do prefer generic function names, overriding common ones (so that people can try what they know and it'll 'just work') where necessary.

**Comment by CarloLucibello** [comment]
let's stick with `load` and `loadgraph` then. This should be ready for merge

File: docs/src/persistence.md [hunk 1]
```diff
@@ -4,17 +4,24 @@ Graphs may be written to I/O streams and files using the `save` function and
...
+save(STDOUT, g)
+save("mygraph.jgz", g, "mygraph", compress=true)
+
+savegraph("mygraph.jgz", g, compress=true)
+
+dg = load("multiplegraphs.jgz") # dictionary of graphs
+dg = load("multiplegraphs.graphml", :graphml)
+dg = load("mygraph.gml", "mygraph", :gml)
+
+g = laoadgraph("mygraph.gml",  :gml)
```
Comment (sbromberger) [inline docs/src/persistence.md]: There's a typo here.

**Pull request merged**